Utils/github_workflow_scripts/close_stale_contrib_branches.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import urllib3
+import inspect
+from blessings import Terminal
+from github import Github, enable_console_debug_logging
+from github.Repository import Repository
+from typing import List
+from dateparser import parse
+from datetime import datetime, timezone
+from utils import timestamped_print
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+print = timestamped_print
+
+
+def get_stale_branch_names_with_contrib(repo: Repository) -> List[str]:  # noqa: E999
+    """Return the list of branches that have the prefix of "contrib/" without open pull requests
+    and that have not been updated for 2 months (stale)
+
+    Args:
+        repo (Repository): The repository whose branches will be searched and listed
+
+    Returns:
+        (List[str]): List of branch names that are stale and have the "contrib/" prefix
+    """
+    now = datetime.now(timezone.min)
```

Comment:
What is this `timezone.min` that you are using?

Reply:
it's 0 - it's basically to set the timezone to GMT.

Reply:
Maybe add that as a comment above it.